test/e2e/kubernetes/scripts/cloud-init-files-validate.sh

```diff
@@ -21,4 +22,8 @@ stat /boot/grub/grub.cfg | grep 'Access: (0400' || exit 1
 # validate grub configuration
 sudo grep "^\s*linux" /boot/grub/grub.cfg | grep 'audit=1' || exit 1
 # validate su configuration
-sudo grep "auth required pam_wheel.so use_uid" /etc/pam.d/su
\ No newline at end of file
+sudo grep "auth required pam_wheel.so use_uid" /etc/pam.d/su
+# validate umask configuration
```

Comment:
should this be in its own validation script? if we don't want to have one script for each small feature maybe we can add a general "validate-cis.sh" script that does all these validations. This file is specific to checking existence of files paved with cloud-init though so it doesn't seem to be the right place to put this validation.

Reply:
I like these functional validations in here because they validate not only that the files are present but that they are having the intended functional effect.